pkg/test/framework/components/cluster/cluster.go

```diff
@@ -102,6 +102,13 @@ func (c Clusters) Remotes(excluded ...Cluster) Clusters {
 	}, exclude(excluded...))
 }
 
+// MeshClusters returns the subset that are not external control plane clusters.
+func (c Clusters) MeshClusters(excluded ...Cluster) Clusters {
```

Comment:
I did sort of like `DataPlaneClusters` as well; not hard-set on that though.

Reply:
I'm fine with either name. I just thought it's more clear to think about it as clusters that are part of the mesh vs. an external control plane cluster, which is just a kubernetes cluster hosting istiod but not itself part of the mesh.